src/components/singlePostBox/SinglePostBox.jsx

```diff
@@ -18,18 +18,18 @@ import useGeneralStore from "../../context/GeneralContext";
 
 const SinglePost = ({ post }) => {
   const [showOptions, setShowOptions] = useState(false);
-  const optRef = useRef();
+  // const optRef = useRef();
   const { currentPage } = useGeneralStore();
   const navigate = /^Posts\//.test(currentPage);
   // const navigate = currentPage.startsWith("Posts/");
 
-  window.addEventListener("click", (e) => {
-    if (!optRef.current.contains(e.target)) {
-      setShowOptions(false);
-    } else {
-      setShowOptions(true);
-    }
-  });
+  // window.addEventListener("click", (e) => {
```

Comment:
Let's remove code comments if they are not in use

Reply:
The page isn't assigned to me. I don't know if whoever has the page still needs it.

Reply:
Cool one it seems we may have inadvertently made few changes  because git is tracking changes in both pages